pallets/phala/src/compute/wrapped_balances.rs

```diff
@@ -143,9 +143,12 @@ pub mod pallet {
 		T: pallet_democracy::Config<Currency = <T as crate::PhalaConfig>::Currency>,
 		T: Config + vault::Config,
 	{
-		fn pre_check(sender: &T::AccountId, collection_id: &CollectionId, nft_id: &NftId) -> bool {
-			if let Some(pid) = base_pool::pallet::PoolCollections::<T>::get(collection_id) {
-				if let Ok(net_value) = Pallet::<T>::get_net_value((*sender).clone()) {
+		fn pre_check(_sender: &T::AccountId, collection_id: &CollectionId, _nft_id: &NftId) -> bool {
+			if base_pool::pallet::PoolCollections::<T>::get(collection_id).is_some() {
+				// Forbid any delegation transfer before delegation nft transfer and sell is fully prepared.
+				// TODO(mingxuan): reopen pre_check function.
+				return false
+				/*if let Ok(net_value) = Pallet::<T>::get_net_value((*sender).clone()) {
```

Comment:
Delete the code. Never comment temporarily kept code in the code base. We are using Git.

Reply:
done